integrations/symfony-bundle.rst

```diff
@@ -16,7 +16,10 @@ Install the HTTPlug bundle with composer and enable it in your AppKernel.php.
 
 .. code-block:: bash
 
-    $ composer require php-http/httplug-bundle
+    $ composer require php-http/httplug-bundle <your client adapter choice>
+
+If you did not install HTTPlug itself, you will also need to specify which HTTP
```

Comment:
Oh, I see. how about:

" If you have not already installed a HTTP client, you need to specify the client when installing the bundle. See [link] for a list of available clients"